src/plugins/dev_tools/public/index.ts

```diff
@@ -6,6 +6,9 @@
  * Side Public License, v 1.
  */
 
+// TODO: https://github.com/elastic/kibana/issues/110892
+/* eslint-disable @kbn/eslint/no_export_all */
+
```

Comment:
This one looks like an error?

Reply:
Very bottom of the file, `export * from './plugin';`